.github/ISSUE_TEMPLATE/release_checklist_template.md

```diff
@@ -15,6 +15,7 @@ Branch setup, steps typically done when creating new version branch
 - [ ] Ensure L10N box has been run (if needed) to do string export
 - [ ] Check for [security advisories](https://github.com/mozilla-mobile/firefox-ios/wiki/Release-Build-Checklist/#security-advisories) 
 - [ ] Tag release in GitHub (Eng task)
+    - [ ] Copy commits with author names into notes section of release
```

Comment:
Confused :), I thought the link to the diff that you started doing was the best way, is this just the contribs names called out?